browser/net/brave_system_request_handler.cc

```diff
@@ -8,6 +8,7 @@
 #include "brave/browser/net/brave_block_safebrowsing_urls.h"
 #include "brave/browser/net/brave_common_static_redirect_network_delegate_helper.h"
 #include "brave/browser/net/brave_static_redirect_network_delegate_helper.h"
+#include "brave/common/brave_services_key.h"
 #include "brave/common/brave_service_key_helper.h"
```

Comment:
should we sync these filenames? make `brave_services_key_helper` or `brave_servic_key`.